controllers/dnsupdate.go

```diff
@@ -103,7 +103,7 @@ func (r *GslbReconciler) gslbDNSEndpoint(gslb *k8gbv1beta1.Gslb) (*externaldns.D
 					log.Info().
 						Str("gslb", gslb.Name).
 						Str("cluster", gslb.Spec.Strategy.PrimaryGeoTag).
-						Str("targets", fmt.Sprintf("%v", finalTargets)).
+						Interface("targets", finalTargets).
```

Comment:
`Interface  - > Strs`? So that we are consistent in this file. I assume the Strs is faster since we know the type and it's not doing the reflection on the argument.

Reply:
@jkremser well spotted, thanks! will amend

Reply:
Done